crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -693,6 +694,16 @@ async fn forward_software_request(
     )])
 }
 
+fn get_smartrest_template(payload: &str) -> Result<String, SmartRestDeserializerError> {
```

Comment:
Template is misleading. What we will get is the message ID, which is defined in the SmartREST template.
```suggestion
fn get_smartrest_message_id(payload: &str) -> Result<String, SmartRestDeserializerError> {
```

Reply:
renamed